isis/src/mro/apps/hideal2pds/main.cpp

```diff
@@ -114,14 +117,47 @@ void IsisMain() {
   pair<double, double> inRange;
   inRange = inputRange(inputCube);
   p.SetInputRange(minmin, maxmax);
-  // output bit type will be 16bit unsigned word
-  p.SetOutputType(Isis::UnsignedWord); 
-  p.SetOutputNull(Isis::NULLU2);         
-  p.SetOutputLrs(Isis::LOW_REPR_SATU2);  
-  p.SetOutputLis(Isis::LOW_INSTR_SATU2); 
-  p.SetOutputHrs(Isis::HIGH_REPR_SATU2); 
-  p.SetOutputHis(Isis::HIGH_INSTR_SATU2);
-  p.SetOutputRange(VALID_MINU2, VALID_MAXU2); 
+
+   int nbits = ui.GetInteger("BITS");
+
+   switch (nbits) {
+
+
+    case 8:
```

Comment:
Indenting on these blocks is off. It looks like everything is single space indented